README.md

```diff
@@ -1,52 +1,57 @@
 - Process changes that affect how the gRPC product is implemented.
 - Breaking changes to the public API (i.e. semver major changes).
 
 ## Process
 
 1. Fork the repo and copy the template [GRFC-TEMPLATE.md](GRFC-TEMPLATE.md).
-1. Rename it to ``$CategoryName-$Summary``, eg.: ``A6-client-retries.md`` (see
-  category definitions below)
+1. Write the RFC.
+1. Rename it to ``$CategoryName##-$Summary``, eg.: ``A6-client-retries.md``
+   (see category definitions below)
    - For language-specific proposals, include the name of the language:
-     ``L##-$Language-$Summary``.  Canonical names: `core`, `cpp`, `csharp`, `go`,
+     ``L##-$Language-$Summary``. Canonical names: `core`, `cpp`, `csharp`, `go`,
      `java`, `node`, `objc`, `php`, `python`, `ruby`.
-1. Write up the RFC.
+   - To determine the number to use for your proposal, view all PRs (open and
+     closed), sorted by creation date ([link](
+     https://github.com/grpc/proposal/pulls?q=is%3Apr+sort%3Acreated-desc)).
+     Find the first _new_ proposal PR of the same type, and use the following
+     number.
 1. Submit a Pull Request.
-1. Someone from gRPC team will be assigned as an APPROVER as part of this
-review. Once the APPROVER is assigned, the OWNER needs to start a discussion on
-[grpc-io](https://groups.google.com/forum/#!forum/grpc-io) and update the PR
-with the discussion link. After this is done, the OWNER should update the gRFC
-to the state of ``In Review``. It is expected that the APPROVER will help the
-OWNER along this process as needed.
-1. For at least a period of 10 business days (the minimum comment period),
-it is expected that the OWNER will respond to the comments and make updates
-to the RFC as new commits to the PR. Through the process, the discussion
-needs to be kept to the designated thread in the mailing list in order to
-avoid splintering conversations. The OWNER is encouraged to solicit as much
-feedback on the proposal as possible during this period.
-PR comments should be limited to formatting and vocabulary.
+1. Someone from the gRPC team will be assigned as an APPROVER as part of this
+review.
+1. For a period of at least 10 business days (the minimum comment period), it is
+expected that the OWNER will respond to the comments and make updates to the RFC
+as new commits to the PR. The OWNER is encouraged to solicit as much feedback on
+the proposal as possible during this period.
 1. If there is consensus as deemed by the APPROVER during the comment period,
-the APPROVER will mark the proposal as final and assign it a gRFC number.
-Once this is assigned (as part of the closure of discussion), the OWNER will
-update the state of the PR as final and submit the PR.
-Commits must not be squashed; the commit history serves as a log of changes
-made to the proposal.
+the APPROVER will approve the PR on GitHub. The PR will then be merged by either
+the OWNER, if possible, or the APPROVER otherwise.
```

Comment:
Should we maybe explicitly say something here like "A proposal being merged indicates that it has been approved", just to make the state clear?

Reply:
It seems pretty clear from the preceding sentence to me...  I added a following sentence to remove any doubt.  PTAL.